packages/solana/src/connectors/walletConnectConnector.ts

```diff
@@ -144,16 +146,56 @@ export class WalletConnectConnector extends BaseConnector implements Connector {
     return { signatures: [{ signature: base58.encode(transaction.serialize()) }] }
   }
 
-  public async sendTransaction(transactionParam: Transaction | VersionedTransaction) {
+  private async _sendTransaction(transactionParam: Transaction | VersionedTransaction) {
```

Comment:
+1

Reply:
The reason is because in wallet adapters [sendTransaction](https://github.com/anza-xyz/wallet-adapter/blob/master/packages/wallets/phantom/src/adapter.ts) returns only signature, so i created  _sendTransaction here which returns tx and signuture, but public sendTransaction has same type as in wallet adapters.